flake.nix

```diff
@@ -0,0 +1,57 @@
+{
+  description = "Status Go Flake";
+
+  nixConfig = {
+    extra-substituters = [ "https://nix-cache.status.im/" ];
+    extra-trusted-public-keys = [ "nix-cache.status.im-1:x/93lOfLU+duPplwMSBR+OlY4+mo+dCN7n0mr4oPwgY=" ];
+    # Some downloads are multiple GB, default is 5 minutes
+    stalled-download-timeout = 3600;
+    connect-timeout = 10;
+    max-jobs = "auto";
+    # Some builds on MacOS have issue with sandbox so they are disabled with __noChroot.
+    sandbox = "relaxed";
+  };
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/df27247e6f3e636c119e2610bf12d38b5e98cc79";
```

Comment:
Would be nice to have a comment on this commit hash.
Whats the aim of this pin? Some tools versions?